diff -up qtwebengine-everywhere-src-6.9.0-rc/src/3rdparty/chromium/third_party/skia/src/opts/SkRasterPipeline_opts.h.2~ qtwebengine-everywhere-src-6.9.0-rc/src/3rdparty/chromium/third_party/skia/src/opts/SkRasterPipeline_opts.h
--- qtwebengine-everywhere-src-6.9.0-rc/src/3rdparty/chromium/third_party/skia/src/opts/SkRasterPipeline_opts.h.2~	2025-03-16 17:41:53.471026606 +0100
+++ qtwebengine-everywhere-src-6.9.0-rc/src/3rdparty/chromium/third_party/skia/src/opts/SkRasterPipeline_opts.h	2025-03-16 17:43:19.874320687 +0100
@@ -1392,7 +1392,7 @@ SI F approx_powf(F x, float y) { return
 #endif
 
 SI F from_half(U16 h) {
-#if defined(SKRP_CPU_NEON) && defined(SK_CPU_ARM64)
+#if 0 && defined(SKRP_CPU_NEON) && defined(SK_CPU_ARM64)
 #if defined(__ARM_FP16_FORMAT_IEEE)
     return vcvt_f32_f16((float16x4_t)h);
 #else
