diff -up qtwebengine-everywhere-src-6.10.0-beta3/src/3rdparty/chromium/base/metrics/histogram_macros.h.4~ qtwebengine-everywhere-src-6.10.0-beta3/src/3rdparty/chromium/base/metrics/histogram_macros.h
diff -up qtwebengine-everywhere-src-6.10.0-beta3/src/3rdparty/chromium/content/browser/webid/federated_auth_request_impl.cc.4~ qtwebengine-everywhere-src-6.10.0-beta3/src/3rdparty/chromium/content/browser/webid/federated_auth_request_impl.cc
--- qtwebengine-everywhere-src-6.10.0-beta3/src/3rdparty/chromium/content/browser/webid/federated_auth_request_impl.cc.4~	2025-08-15 20:30:00.000000000 +0200
+++ qtwebengine-everywhere-src-6.10.0-beta3/src/3rdparty/chromium/content/browser/webid/federated_auth_request_impl.cc	2025-08-23 22:51:04.957978442 +0200
@@ -2501,7 +2501,7 @@ void FederatedAuthRequestImpl::ShowModal
   // dialog?
   dialog_type_ = dialog_type;
   config_url_ = idp_config_url;
-  UMA_HISTOGRAM_ENUMERATION("Blink.FedCm.Popup.DialogType", dialog_type_);
+  //CLANG21 UMA_HISTOGRAM_ENUMERATION("Blink.FedCm.Popup.DialogType", dialog_type_);
 
   WebContents* web_contents = request_dialog_controller_->ShowModalDialog(
       url_to_show, rp_mode_,
diff -up qtwebengine-everywhere-src-6.10.0-beta3/src/3rdparty/chromium/sandbox/policy/linux/sandbox_linux.cc.4~ qtwebengine-everywhere-src-6.10.0-beta3/src/3rdparty/chromium/sandbox/policy/linux/sandbox_linux.cc
--- qtwebengine-everywhere-src-6.10.0-beta3/src/3rdparty/chromium/sandbox/policy/linux/sandbox_linux.cc.4~	2025-08-15 20:30:00.000000000 +0200
+++ qtwebengine-everywhere-src-6.10.0-beta3/src/3rdparty/chromium/sandbox/policy/linux/sandbox_linux.cc	2025-08-23 22:51:04.958231659 +0200
@@ -697,7 +697,8 @@ void SandboxLinux::ReportLandlockStatus(
     landlock_state = LandlockState::kEnabled;
   }
 
-  UMA_HISTOGRAM_ENUMERATION("Security.Sandbox.LandlockState", landlock_state);
+  // FIXME causes build failure with clang 21
+  //UMA_HISTOGRAM_ENUMERATION("Security.Sandbox.LandlockState", landlock_state);
 }
 
 }  // namespace policy
