diff -up qtwebengine-everywhere-src-6.7.0/src/3rdparty/chromium/base/strings/utf_string_conversion_utils.h.omv~ qtwebengine-everywhere-src-6.7.0/src/3rdparty/chromium/base/strings/utf_string_conversion_utils.h
--- qtwebengine-everywhere-src-6.7.0/src/3rdparty/chromium/base/strings/utf_string_conversion_utils.h.omv~	2024-05-09 02:03:46.643336861 +0200
+++ qtwebengine-everywhere-src-6.7.0/src/3rdparty/chromium/base/strings/utf_string_conversion_utils.h	2024-05-09 02:04:05.947534583 +0200
@@ -12,6 +12,7 @@
 #include <stdint.h>
 
 #include <string>
+#include <limits>
 
 #include "base/base_export.h"
 #include "base/third_party/icu/icu_utf.h"
diff -up qtwebengine-everywhere-src-6.7.0/src/3rdparty/chromium/components/os_crypt/sync/kwallet_dbus.h.omv~ qtwebengine-everywhere-src-6.7.0/src/3rdparty/chromium/components/os_crypt/sync/kwallet_dbus.h
--- qtwebengine-everywhere-src-6.7.0/src/3rdparty/chromium/components/os_crypt/sync/kwallet_dbus.h.omv~	2024-05-09 01:57:13.435240697 +0200
+++ qtwebengine-everywhere-src-6.7.0/src/3rdparty/chromium/components/os_crypt/sync/kwallet_dbus.h	2024-05-09 01:57:19.867309072 +0200
@@ -7,6 +7,7 @@
 
 #include <string>
 #include <vector>
+#include <cstdint>
 
 #include "base/component_export.h"
 #include "base/memory/scoped_refptr.h"
diff -up qtwebengine-everywhere-src-6.7.0/src/3rdparty/chromium/content/common/service_worker/service_worker_router_evaluator.cc.omv~ qtwebengine-everywhere-src-6.7.0/src/3rdparty/chromium/content/common/service_worker/service_worker_router_evaluator.cc
--- qtwebengine-everywhere-src-6.7.0/src/3rdparty/chromium/content/common/service_worker/service_worker_router_evaluator.cc.omv~	2024-05-09 02:28:00.969388515 +0200
+++ qtwebengine-everywhere-src-6.7.0/src/3rdparty/chromium/content/common/service_worker/service_worker_router_evaluator.cc	2024-05-09 02:28:06.801441078 +0200
@@ -446,7 +446,7 @@ ServiceWorkerRouterEvaluator::~ServiceWo
 
 void ServiceWorkerRouterEvaluator::Compile() {
   for (const auto& r : rules_.rules) {
-    std::unique_ptr<RouterRule> rule = absl::make_unique<RouterRule>();
+    std::unique_ptr<RouterRule> rule = std::make_unique<RouterRule>();
     if (!rule->SetRule(r)) {
       return;
     }
diff -up qtwebengine-everywhere-src-6.7.0/src/3rdparty/chromium/services/metrics/public/cpp/mojo_ukm_recorder.cc.omv~ qtwebengine-everywhere-src-6.7.0/src/3rdparty/chromium/services/metrics/public/cpp/mojo_ukm_recorder.cc
--- qtwebengine-everywhere-src-6.7.0/src/3rdparty/chromium/services/metrics/public/cpp/mojo_ukm_recorder.cc.omv~	2024-05-09 03:07:16.986406334 +0200
+++ qtwebengine-everywhere-src-6.7.0/src/3rdparty/chromium/services/metrics/public/cpp/mojo_ukm_recorder.cc	2024-05-09 03:07:20.230444154 +0200
@@ -17,7 +17,7 @@ namespace ukm {
 
 std::unique_ptr<ukm::MojoUkmRecorder> MojoUkmRecorder::Create(
     mojom::UkmRecorderFactory& factory) {
-  return absl::WrapUnique(new MojoUkmRecorder(factory));
+  return base::WrapUnique(new MojoUkmRecorder(factory));
 }
 
 MojoUkmRecorder::MojoUkmRecorder(mojom::UkmRecorderFactory& factory) {
