diff -up qtwebengine-everywhere-src-6.2.2/src/3rdparty/chromium/third_party/harfbuzz-ng/src/src/hb-subset-cff1.cc.omv~ qtwebengine-everywhere-src-6.2.2/src/3rdparty/chromium/third_party/harfbuzz-ng/src/src/hb-subset-cff1.cc
--- qtwebengine-everywhere-src-6.2.2/src/3rdparty/chromium/third_party/harfbuzz-ng/src/src/hb-subset-cff1.cc.omv~	2022-01-09 23:52:18.991014134 +0100
+++ qtwebengine-everywhere-src-6.2.2/src/3rdparty/chromium/third_party/harfbuzz-ng/src/src/hb-subset-cff1.cc	2022-01-09 23:55:01.888478524 +0100
@@ -402,7 +402,7 @@ struct cff_subset_plan {
   void plan_subset_encoding (const OT::cff1::accelerator_subset_t &acc, hb_subset_plan_t *plan)
   {
     const Encoding *encoding = acc.encoding;
-    unsigned int  size0, size1, supp_size;
+    unsigned int  size0, size1;
     hb_codepoint_t  code, last_code = CFF_UNDEF_CODE;
     hb_vector_t<hb_codepoint_t> supp_codes;
 
@@ -412,11 +412,12 @@ struct cff_subset_plan {
       return;
     }
 
-    supp_size = 0;
     supp_codes.init ();
 
     subset_enc_num_codes = plan->num_output_glyphs () - 1;
     unsigned int glyph;
+#pragma GCC diagnostic ignored "-Wunused-but-set-variable"
+    unsigned int supp_size = 0;
     for (glyph = 1; glyph < plan->num_output_glyphs (); glyph++)
     {
       hb_codepoint_t  old_glyph;
